catalog/spatial/registry/registry-app/src/main/resources/features.xml

```diff
@@ -28,7 +28,7 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
-    <feature name="registry-app" install="manual" version="${project.version}"
+    <feature name="registry-app" install="auto" version="${project.version}"
```

Comment:
I turned it on because I was under the impression that when we merge back into codice/ddf that the registry app would need to be on by default. The changes in this PR definitely require that is on by default since the sources tab still (for the moment) lives in the catalog-app yet the poller has moved out of that bundle. This will be settled later today hopefully.